docs/website/docs/plus/features/projects.md

```diff
@@ -0,0 +1,194 @@
+
+1. [Using a declarative `dlt.yml` file](#the-dlt-manifest-file-dltyml) to define sources, destinations, pipelines, and transformations.
+2. Configuring [different profiles](../core-concepts/profiles.md)  for various use cases and environments. 
+3. Ensuring data quality by defining tests with [dlt+ tests utils](./quality/tests.md).
+4. Packaging your project as a Python package and distributing it via PyPI or git repository.
+
+This structured approach allows teams to work efficiently while maintaining flexibility and control over their data workflows.
+
+## Project structure
+  
+A dlt+ Project has the following general structure:
+```text
+.
+├── dlt_example_project
+│    ├── .dlt/                 # folder containg dlt configrations and profile settings
+│    │   ├── config.toml
+│    │   ├── dev.secrets.toml  # credentials for access profile 'dev'
+│    │   └── secrets.toml      
+│    ├── sources/              # modules containing the source code for sources 
+│    │   └── github.py         # source code for a github source   
+│    ├── transformations/      # modules containing the source code for transformations 
+│    ├── .gitignore
+│    └── dlt.yml               # the main dlt manifest file
+└── pyproject.toml             # the python manifest file for the package
+```
+
+### The dlt manifest file (dlt.yml)
+
+The main component of a dlt+ Project is the dlt manifest file (`dlt.yml`). It marks the root of your project and contains the main configurations. Here you can declare all of your data platform entities in a yaml format. It contains the following sections:
+
+#### Sources
+
+This section lets you define sources either declaratively or by referencing an implementation from a python module inside `sources/`. In the example below, two sources are declared:  
+1. a dlt REST API source whose parameters are passed within the manifest
+2. a github source whose source code inside `sources/github.py` is referenced
+  
+```yaml
+sources:
+  pokemon:
+    type: dlt.sources.rest_api.rest_api
+    client:
+      base_url: https://pokeapi.co/api/v2/
+    resource_defaults:
+      endpoint:
+        params:
+          limit: 1000
+    resources:
+      - pokemon
+      - berry
+
+  github:
+    type: dlt_example_project.github.source
+```
+#### Destinations  
+  
+The destinations section defines dlt destinations in a similar way to how you would define them in a pure Python dlt project. As with sources, you can also create a `destinations/` folder and reference custom implementations of destinations inside it.
+  
+```yaml  
+destinations:
+    duckdb:
+        type: duckdb
+```
+
+#### Pipelines 
+
+Pipelines can be used to load data from sources to destinations. The pipeline defined below loads data from the github source to a dataset named "github_events_dataset" inside the duckdb destination.
+
+```yaml
+  github_pipeline:
+    source: github
+    destination: duckdb
+    dataset_name: github_events_dataset
```

Comment:
```suggestion
github_pipeline:
  source: github
  destination: duckdb
  dataset_name: github_events_dataset
```

Reply:
fixed in https://github.com/dlt-hub/dlt/pull/2325